templates/validate-h-card.html.php

```diff
@@ -93,6 +93,8 @@
 			<div class="preview-h-card preview-block">
 				<?php if (Mf2\hasProp($hCard, 'photo')): ?>
 				<img class="photo-block" src="<?= Mf2\getProp($hCard, 'photo')?>" alt="" />
+				<?php if (Mf2\hasProp($hCard, 'logo')): ?>
```

Comment:
This `if` was inserted in the middle of the above `if`, so now there is a syntax error. This logic needs to be fixed since the site is currently broken in this state.